### PR TITLE
More UI Customizations and filters

### PR DESCRIPTION
**Pull request by willdeberry**
Create the ability to filter Build History and Job Status individually. This
allows the user to create different regexs for each column.

For Example:
* Use the build history as an overall timeline
* Filter the job statuses to your most important builds that should always be
green

### CODE REVIEW

**Review by a-shevtsov — CHANGES_REQUESTED** [review]
This is a not very common case to break backward compatibility. What I would do is to leave existing filter and apply it both build history and job statuses by default (as it is now) but add another checkbox ( 'Different regex for Build History' for example) for those who needs this option.

File: src/main/java/tech/andrey/jenkins/missioncontrol/MissionControlView.java [hunk 1]
```diff
@@ -383,4 +398,58 @@ public JobStatus(String jobName, String status) {
...
+            if ("SUCCESS".equalsIgnoreCase(s)) {
+                return 5;
+            }
+            if ("NOTBUILT".equalsIgnoreCase(s)) {
+                return 6;
+            }
+            if ("DISABLED".equalsIgnoreCase(s)) {
+                return 7;
+            }
+            return 0;
```
Comment (a-shevtsov) [inline src/main/java/tech/andrey/jenkins/missioncontrol/MissionControlView.java]: Please replace this with a static array/dictionary with keys in lower or upper case and convert status to that case when retrieving sort order value.

**Review by a-shevtsov — CHANGES_REQUESTED** [review]
Good option, just need to make one change (see the comment).

**Comment by willdeberry** [comment]
Anything else need to be adjusted? Let me know, thanks!

File: src/main/java/tech/andrey/jenkins/missioncontrol/MissionControlView.java (hunk 1 above)
Comment (willdeberry) [reply src/main/java/tech/andrey/jenkins/missioncontrol/MissionControlView.java]: Done

**Review by a-shevtsov — APPROVED** [review]

**Comment by a-shevtsov** [comment]
Thanks for making these changes. I'm really sorry I could not review this earlier.

**Pull request merged**